schema-registry/src/types.rs

```diff
@@ -79,3 +80,35 @@ impl VersionedUuid {
         }
     }
 }
+
+// Import export
+#[derive(Default, Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct DbExport {
+    pub schemas: HashMap<Uuid, vertices::Schema>,
+    pub definitions: HashMap<Uuid, vertices::Definition>,
+    pub views: HashMap<Uuid, vertices::View>,
+    pub def_edges: Vec<edges::SchemaDefinition>,
+    pub view_edges: Vec<edges::SchemaView>,
+}
+
+fn get_vertex_property_or<T: DeserializeOwned>(
```

Comment:
```suggestion
fn extract_vertex_property<T: DeserializeOwned>(
```
To imply that we are removing the desired property.